### PR TITLE
Update: Completed Avtar and Badges

### PR DESCRIPTION
**Pull request by HeyNitin**
Have created Avtar and Badges. 

image
image

### CODE REVIEW

File: Components/Badges/badges.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
+@import "../Avtars/avtars.css";
+
+
+.badge{
+    display: inline-block;
+    position: relative;
+    margin: 1rem;
+    margin-bottom: 5rem;
```
Comment (MaruthiPothuganti) [inline Components/Badges/badges.css]: Suggestion: Margin is repeated

Comment (MaruthiPothuganti) [reply Components/Badges/badges.css]: can use a shorthand propert

**Review by MaruthiPothuganti — COMMENTED** [review]
recommended changes: use of short hand properties

**Review by Hardiegogo — COMMENTED** [review]
Looks good to me. You can merge it after making changes to margin property.(use shorthand)

**Pull request merged**